src/internal/connector/data_collections.go

```diff
@@ -46,6 +46,10 @@ func (gc *GraphConnector) ProduceBackupCollections(
 		diagnostics.Index("service", sels.Service.String()))
 	defer end()
 
+	// Limit the max number of active requests to graph
```

Comment:
nit: make clear this is not concurrency control across collections (yet?)

```suggestion
	// Limit the max number of active requests to graph from this collection
```